plan/planbuilder.go

```diff
@@ -117,6 +120,31 @@ func (info *tableHintInfo) ifPreferINLJ(tableNames ...*model.CIStr) bool {
 	return false
 }
 
+// clauseCode indicates in which clause the column is currently.
+type clauseCode int
+
+const (
+	unknowClause  clauseCode = 0
```

Comment:
1. `unknowClause  clauseCode = iota`
2. how about let all clause code have a common prefix, like "clauseCodeXXX" ?

Reply:
For my part, the current variable names may be better.